content/integrations/hipchat.md

```diff
@@ -28,7 +28,10 @@ Tick the checkbox if you want to be notified for every comment, in all configure
 
 You can now share graphs or send alerts to HipChat rooms using the syntax @hipchat-chat_name
 
-**NOTICE** : If your chat name contains special characters such as commas or brackets, they'll be escaped when sending notifications with the @ handle. You shouldn't have to worry about that, the autocomplete box automatically takes care of escaping chat names.
+**NOTICE** : 
```

Comment:
Instead of **NOTICE**, let's wrap this in a warning block, e.g.

<div class="alert alert-warning">
If you are using a hip chat API v1 token...
...
</div>